src/pages/ReimbursementAccount/BankAccountManualStep.js

```diff
@@ -0,0 +1,171 @@
+import _ from 'underscore';
+import React from 'react';
+import {Image, View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import CONST from '../../CONST';
+import * as BankAccounts from '../../libs/actions/BankAccounts';
+import Navigation from '../../libs/Navigation/Navigation';
+import Text from '../../components/Text';
+import TextInput from '../../components/TextInput';
+import styles from '../../styles/styles';
+import CheckboxWithLabel from '../../components/CheckboxWithLabel';
+import TextLink from '../../components/TextLink';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import * as ValidationUtils from '../../libs/ValidationUtils';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as ReimbursementAccount from '../../libs/actions/ReimbursementAccount';
+import exampleCheckImage from './exampleCheckImage';
+import ReimbursementAccountForm from './ReimbursementAccountForm';
+import * as ReimbursementAccountUtils from '../../libs/ReimbursementAccountUtils';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+class BankAccountManualStep extends React.Component {
```

Comment:
Heads up @Gonals, we are splitting the manual step into its own component. Maybe for https://github.com/Expensify/App/pull/10987, you can branch off this PR.